src/commands/openai.ts

```diff
@@ -15,6 +15,11 @@ export const openai = cli()
     "-k, --apiKey <apiKey>",
     "The OpenAI API key. Alternatively use OPENAI_API_KEY environment variable"
   )
+  .option(
+    "-s, --server <server>",
```

Comment:
Might it be better to align it more directly to 'base URL' type naming?

Reply:
Good point, changed to `--basURL` to match OpenAI naming:

https://github.com/openai/openai-node/blob/6ae19ce08eea4f6b3c4865861a6cce09d403cac8/src/index.ts#L106